go/private/rules/wrappers.bzl

```diff
@@ -25,7 +25,11 @@ _CGO_ATTRS = {
     "srcs": None,
     "cdeps": [],
     "copts": [],
+    "cxxopts": [],
+    "cppopts": [],
     "clinkopts": [],
+    "objc": False,
+    "objcopts": {},
```

Comment:
In the future, if we're able to compile C/C++/ObjC code directly with actions, we'll convert `go_library`, `go_binary`, `go_test` to regular rules (no wrapper macros). We won't be able to have an attribute like `objcopts`, so this will definitely break.

Maybe it could be a series of attributes with an `objc_` prefix instead? `objc_sdk_frameworks` for example? Please only include attributes for things that `go build` would pick up though. For example, we should not support `xibs` here.

Reply:
I'm fine with `objc_` prefix. Good catch for the `xibs` parameter.